tslint/noQueryRendererImportRule.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+
+const Lint = require("tslint")
+const ts = require("typescript")
+
+class Rule extends Lint.Rules.AbstractRule {
```

Comment:
This approach works, but is purely text-based. Someone could have a variable called "QueryRenderer" and trip this up by accident (unlikely to happen, just as an example).

I think this rule lends itself to working with the TypeScript Abstract Syntax Tree. You could look for only the `import` statements in a file instead of scanning its text contents. Here's an example of how TSLint does this with its rule to avoid duplicated import statements:

https://github.com/palantir/tslint/blob/70e24bcb2379e0e60f7d3bfa91fc0c1b6c0c8a9c/src/rules/noDuplicateImportsRule.ts#L101-L111